programs/gamma/src/states/global_reward_info.rs

```diff
@@ -0,0 +1,143 @@
+use crate::{borsh::maybestd::collections::VecDeque, error::GammaError};
+use anchor_lang::prelude::*;
+
+use super::RewardInfo;
+
+pub const MAX_REWARDS: usize = 3;
+
+#[account]
+pub struct GlobalRewardInfo {
+    // This contains the 3 active boosted rewards, i.e. all rewards that are not fully distributed
+    // And the current time maybe exceeds the end time of the last boosted reward
+    // There is never a proper endtime of the rewards we can even have active boosted rewards if they are not fully distributed yet.
+    // Any reward that is not started yet is also consider active.
+    pub active_boosted_reward_info: [Pubkey; MAX_REWARDS],
```

Comment:
Think it's more reasonable to combine the two `active_boosted_reward_info` and `start_times` structures into one. 

Leaving a single `active_boosted_reward_info` field with type `[Reward; MAX_REWARDS]` where `Reward` is defined as:

```rust
struct Reward {
    pubkey: Pubkey,
    start_time: Option<u64>
}
```


```suggestion
    pub active_boosted_reward_info: [Reward; MAX_REWARDS],
```